oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBuilder.java

```diff
@@ -608,13 +621,17 @@ public long getPrevDocumentCacheSize() {
         return memoryCacheSize * prevDocCachePercentage / 100;
     }
 
+    public long getPrevNoPropCacheSize() {
+        return memoryCacheSize * prevNoPropCachePercentage / 100;
+    }
+
     public long getChildrenCacheSize() {
         return memoryCacheSize * childrenCachePercentage / 100;
     }
 
     public long getDocumentCacheSize() {
         return memoryCacheSize - getNodeCacheSize() - getPrevDocumentCacheSize() - getChildrenCacheSize()
-                - getDiffCacheSize();
+                - getDiffCacheSize() - getPrevNoPropCacheSize();
```

Comment:
same as above, value should depend upon prevNoPropCache feature toggle.

Reply:
indeed

Reply:
addressed in https://github.com/apache/jackrabbit-oak/pull/1779/commits/faa44dffb661a8776dcef7d335e239096e17fce4 - also introduced an actual config (with default 1). toggle overwrites config of course.